code/__defines/rust_g.dm

```diff
@@ -188,10 +213,10 @@
 	else
 		CRASH(output["content"])
 
-#define rustg_udp_send(addr, text) LIBCALL(RUST_G, "udp_send")(addr, text)
+#define rustg_udp_send(addr, text) call(RUST_G, "udp_send")(addr, text)
```

Comment:
That looks like its going to cause issues with 515 (call vs ext_call vs RUSTG_CALL)

Reply:
that's a problem with our rust-g fork then, because this is the autogenerated file from rust-g